core/lib/presentation/extensions/color_extension.dart

```diff
@@ -236,6 +236,7 @@ extension AppColor on Color {
   static const colorContactViewClearFilterButton = Color(0x001C3D0D);
   static const steelGrayA540 = Color(0xFF55687D);
   static const blue700 = Color(0xFF208BFF);
+  static const colorShowAllAttachmentsButton = Color(0xFF818C99);
```

Comment:
Looks like we already have this color code, please reuse it, avoid creating unnecessary ones

Reply:
Check if this color code already has a generic name, not specific to any one. If so, reuse it, if not, leave it as is.